### PR TITLE
part 2 of the script that will import stuck reportback submissions.

### PR DESCRIPTION
**Pull request by rapala61**
## How to Test
- 👀 
- Update your .env to have all production variables
- Run `npm i`
- Run `NODE_ENV=test npm start` from `odoyle-scripts/gambit-stuck-submissions-import/`
- You should see all stuck reportbacks being re-posted to the included test API.

### CODE REVIEW

File: gambit-stuck-submissions-import/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#### Production 🚨
+Run `NODE_ENV=production npm start`
+- ⚠️: Will re post reportbacks using the base url and gambit API key stored in your `.env` file. Potentially impacting real, production users.
+
+# Some things to consider
+- This script will **only** re-post reportbacks **directly** to Gambit that:
+  - belong to currently running campaigns.
+  - **Do not** have a `submitted_at` property.
+  - **Have** a `failed_at` property.
+  - It's user **has** a `mobilecommons_id` property.
```
Comment (aaronschachter) [inline gambit-stuck-submissions-import/README.md]: The grammar police officer in me can't resist pointing out that we want `its user` over `it's user` 🚔

**Review by aaronschachter — APPROVED** [review]
I've run this locally for `development` and `test`, and see the logs for `info: Successfully sent request. Status: 200. Data: {"message":"It's all good! 😁"}`. 

Not a blocker, but are we able to log the phone numbers / keywords to sanity check the db / user's profile in Mobile Commons for a given success or error?

File: gambit-stuck-submissions-import/README.md (hunk 1 above)
Comment (rapala61) [reply gambit-stuck-submissions-import/README.md]: I usually sacrifice grammar mental units past 🛏⏰😅

**Comment by rapala61** [comment]
@aaronschachter I made some changes to fix the typo and make the logs more useful. Give it a shot and let me know.

**Review by aaronschachter — APPROVED** [review]
Awesome! 💯

**Pull request merged**